trackpy/linking/linking.py

```diff
@@ -96,15 +92,23 @@ def link(f, search_range, pos_columns=None, t_column='frame', **kwargs):
         algorithm used to resolve subnetworks of nearby particles
         'auto' uses numba if available
         'drop' causes particles in subnetworks to go unlinked
+    neighbor_strategy : {'KDTree', 'BTree'}
+        algorithm used to identify nearby features. Default 'KDTree'.
+    to_eucl : function, optional
```

Comment:
Yes this `to_eucl` might be done as preprosessing as well. We do need the implementation anyway for the anisotropic-pixel size rescaling. Might as well expose it to the user.